db/seeds.rb

```diff
@@ -232,7 +232,7 @@
 
 # we'll create 5 patients with pledges at different times
 (1..5).each do |patient_number|
-  pledge_time = 
+  pledge_time =
```

Comment:
Bleh, I think I missed a spot on that other branch and accidentally screwed this up. Would you mind removing this line? Sorry about that.